lib/widgets/navigation_drawer.dart

```diff
@@ -64,23 +70,25 @@ class NavigationDrawer extends StatelessWidget {
   Widget build(BuildContext context) {
     final themeData = Theme.of(context);
     final userProfileBloc = AppBlocsProvider.of<UserProfileBloc>(context);
+    final colorScheme = locator.get<AppConfigScheme>().drawerItemConfig;
 
     List<Widget> children = [
-      _walletDrawerHeader(userProfileBloc, _avatar),
-      Padding(padding: EdgeInsets.only(top: 16)),
+      _walletDrawerHeader(userProfileBloc, _avatar, colorScheme),
+      const Padding(padding: EdgeInsets.only(top: 16)),
     ];
     _drawerGroupedItems.forEach((groupItems) {
       children.addAll(_createDrawerGroupWidgets(
         groupItems,
         context,
+        colorScheme,
         _drawerGroupedItems.indexOf(groupItems),
-        withDivider: children.length > 0 && groupItems.withDivider,
+        withDivider: children.isNotEmpty && groupItems.withDivider,
```

Comment:
can this still be 0, though?

Reply:
No, the logic is next drawing divider except of the last item.